Feature/update circle ci

carlosesilva opened the pull request.

Commit:
change phpunit to 5.7

wait until wp is fully compatible with phpunit 6.1

Commit:
remove code sniffer tests

Commit:
test on different php versions

Commit:
add circleci status shield to readme

Commit:
update readme with new repo name

desrosj commented:
@carlosesilva this looks good. Only thought is we maybe could add a test for `trunk` with the environment version (7.1.x). But 👍

Commit:
add test for trunk on PHP7.1 for both ci

remove WP_MULTISITE variable from the matrix declaration in travis since the script tests for both single and multisite

carlosesilva commented:
Thanks @desrosj !
I have added the test for trunk on both travis and circle.
I also removed the WP_MULTISITE variable from the matrix declaration in travis since the script runs phpunit on both single and multisite already.
